controllers/registration.js

```diff
@@ -1,173 +1,219 @@
-
 const express = require('express');
+const DeviceDetector = require('device-detector-js');
+
 const router = express.Router();
 const api = require('../api');
 const { cookieDomain } = require('../helpers/authentication');
 
+const deviceDetector = new DeviceDetector();
```

Comment:
naming würde ich noch anpassen. zwei variablen welche sich nur in Groß/Kleinschreibung unterscheiden kann schnell zu verwirrung führen.
`deviceDetector` vs `DeviceDetector`